source/app.d

```diff
@@ -55,6 +55,21 @@ shared static this()
 	immutable certPath = redhatCA.exists ? redhatCA : debianCA;
 }
 
+// generate dummy data for e.g. Heroku's preview apps
+void defaultInit(UserManController userMan, DubRegistry registry)
+{
+	if (environment.get("GENERATE_DEFAULT_DATA", "0") == "1" &&
+		registry.getPackageDump().empty)
+	{
+		logInfo("'GENERATE_DEFAULT_DATA' is set and an empty database has been detected. Inserting dummy data.");
+		auto userId = userMan.registerUser("dummy@dummy.org", "dummyUser",
+			"Dummy User", "test1234");
```

Comment:
I don't think there's anything to be gained by exposing the dummy password here.
The worst one can due is to register a few packages (or delete them), but the review app is deleted after five days (or a merge) anyhow.
In case this really becomes a problem, we can always use an auto-generated password here.